sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/Paths.java

```diff
@@ -8,6 +8,8 @@
  */
 public class Paths {
     static final String ROOT = "/";
+    static final Character ROOT_CHAR = '/';
+    static final Character ESCAPE_CHAR = '\\';
```

Comment:
Any specific reason why are we not using primitive types here -> `char` ?

Reply:
Because using `Character` class here will result in auto-boxing later in your code below.